cli.py

```diff
@@ -60,7 +60,7 @@
         (re.compile('000413(26|2E|3A|3C|52)[0-9A-F]{4}'), 'snom370'),
         (re.compile('000413(30|56)[0-9A-F]{4}'), 'snomm9'),
         (re.compile('000413(32|55)[0-9A-F]{4}'), 'snomMP'),
-        (re.compile('00041361[0-9A-F]{4}|00087B(08|09|0B)[0-9A-F]{4}'), 'snomM700'),
+        (re.compile('00041361|63[0-9A-F]{4}|00087B(08|09|0B)[0-9A-F]{4}'), 'snomM700'),
```

Comment:
No, please put braces around the "or" clause how it's done already for MAC range 32 i.e.